File: Palindromic Array - GFG/palindromic-array.java
//{ Driver Code Starts
import java.util.*;
class PalindromicArray{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		int t = sc.nextInt();
		while(t-->0)
		{
			int n = sc.nextInt();
			int[] a = new int[n];
			for(int i = 0 ;i < n; i++)
				a[i]=sc.nextInt();
			GfG g = new GfG();
			System.out.println(g.palinArray(a , n));
		}
	}
}
// } Driver Code Ends


/*Complete the Function below*/
class GfG
{
	public static int palinArray(int[] a, int n)
           {
                  //add code here.
                  for(int i=0;i<n;i++)
                  {
                      int ans=0;
                      int t=a[i];
                      while(t!=0){
                          int d=t%10;
                          ans=ans*10+d;
                          t/=10;
                      }
                      if(a[i]!=ans)
                        return 0;
                  }
                  return 1;
           }
}

File: Palindromic Array - GFG/palindromic-array.cpp
//{ Driver Code Starts
#include<iostream>
#include<string.h>
using namespace std;


// } Driver Code Ends
/*Complete the function below*/

class Solution {
public:
    int PalinArray(int a[], int n)
    {
    	// code here
    
    	for(int i=0;i<n;i++){
    	    int ans=0;
    	    int temp=a[i];
    	    while(temp>0){
    	        int d=temp%10;
    	        ans=ans*10+d;
    	        temp/=10;
    	    }
    	    if(ans!=a[i])
    	      return 0;
    	}
    	return 1;
    }
};

//{ Driver Code Starts.

int main()
{
	int t;
	cin>>t;
	while(t--)
	{
		int n;
		cin>>n;
		int a[n];
		for(int i = 0; i < n; i++)
			cin>>a[i];
		Solution obj;
		cout<<obj.PalinArray(a,n)<<endl;
	}
}
// } Driver Code Ends